src/modules/Dropdown/Dropdown.js

```diff
@@ -403,7 +403,9 @@ export default class Dropdown extends Component {
     if (!prevState.focus && this.state.focus) {
       debug('dropdown focused')
       if (!this.isMouseDown) {
+        const { openOnFocus } = this.props
         debug('mouse is not down, opening')
+        if (!openOnFocus) return
```

Comment:
Small syntactical change request: `if (openOnFocus) this.open()`, since there is an explicit 1:1 correlation between the condition and the action in this case.